docs/INSTALL_cn.md

```diff
@@ -7,21 +7,21 @@
 ## Linux
 
   ### 安装 **OpenVINO™ integration with TensorFlow** PyPi 发布版
-  * 包括 Intel<sup>®</sup> OpenVINO™ 2022.1.0 版的预建库，用户无需单独安装 OpenVINO™。
+  * 包括 Intel<sup>®</sup> OpenVINO™ 2022.2.0 版的预建库，用户无需单独安装 OpenVINO™。
   * 支持 Intel<sup>®</sup> CPU、Intel<sup>®</sup> 集成 GPU 和 Intel<sup>®</sup> Movidius™ 视觉处理单元 (VPU)，但不支持 VAD-M。
 
         pip3 install -U pip
-        pip3 install tensorflow==2.9.1
-        pip3 install openvino-tensorflow==2.1.0
-    openvino-tensorflow PyPi 安装包对tensorflow的补丁版交叉兼容。例如 openvino-tensorflow 针对TF 2.9.1 的模块依然可以在像 TF 2.9.2 和 2.9.3 这样的补丁版本上运行。
+        pip3 install tensorflow==2.9.2
+        pip3 install openvino-tensorflow==2.2.0
+    openvino-tensorflow PyPi 安装包对tensorflow的补丁版交叉兼容。例如 openvino-tensorflow 针对TF 2.9.2 的模块依然可以在像 TF 2.9.2 和 2.9.3 这样的补丁版本上运行。
 <br/>
 
   ### 安装 **OpenVINO™ integration with TensorFlow** PyPi 发布版与独立安装Intel® OpenVINO™ 发布版以支持VAD-M
-    * 兼容 Intel<sup>®</sup> OpenVINO™ 2022.1.0版本
+    * 兼容 Intel<sup>®</sup> OpenVINO™ 2022.2.0版本
     * 支持 Intel<sup>®</sup> Movidius™ (VAD-M) 的视觉加速器设计 同时支持 Intel<sup>®</sup> CPU、Intel<sup>®</sup> 集成 GPU、Intel<sup>®</sup> Movidius™ 视觉处理单元 (VPU)。 
     * 使用方法：
     1. 按照上述方法从PyPi安装tensorflow 和 openvino-tensorflow。
-    2. 下载安装Intel<sup>®</sup> OpenVINO™ 2022.1.0发布版，一并安装其依赖([https://software.intel.com/en-us/openvino-toolkit/download](https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit/download.html))。
+    2. 下载安装Intel<sup>®</sup> OpenVINO™ 2022.2.0发布版，一并安装其依赖([https://software.intel.com/en-us/openvino-toolkit/download](https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit/download.html))。
     3. 初始化Intel<sup>®</sup> OpenVINO™, 可以运行位于<code>\<openvino\_install\_directory\>\/bin</code> 的命令脚本`setupvars.sh` 。该步骤需要在运行openvino-tensorflow做推理的同一个环境下执行。
```

Comment:
Similarly here too. <openvino_root_directory> instead of <openvino_root_directory>/bin.

Reply:
Will take this change as a part of another docs PR